reconcile/aus/base.py

```
@@ -1151,14 +1153,89 @@ def _calculate_node_pool_diffs(
+) -> list[str]:
+    """Fetch available_channels via individual cluster GET.
+
+    The cluster list API does not populate available_channels, so we
+    need a per-cluster GET when checking for Y-stream channel switches.
+    """
+    try:
+        data = ocm_api.get(
+            api_path=f"/api/clusters_mgmt/v1/clusters/{cluster.id}"
+        )
+        return data.get("version", {}).get("available_channels") or []
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)